examples/randomwalks/ppo_randomwalks.py

```diff
@@ -18,10 +18,14 @@ def main(hparams={}):
 
     trlx.train(
         "CarperAI/randomwalks",
-        reward_fn=lambda samples, **kwargs: metric_fn(samples)["optimality"],
+        # An "optimality" reward function is used, with scores in [0,1]
+        # depending on how close the path is to the shortest possible path.
+        reward_fn=lambda samples, p, o: metric_fn(samples, p, o)["optimality"],
```

Comment:
Revert this. RL Trainer calls the `reward_fn` with the keyword args `samples, prompts, outputs` and this breaks the call.